addons/sale_project/models/sale_order.py

```diff
@@ -201,7 +201,7 @@ def _timesheet_create_project_prepare_values(self):
         return {
             'name': '%s - %s' % (self.order_id.client_order_ref, self.order_id.name) if self.order_id.client_order_ref else self.order_id.name,
             'analytic_account_id': account.id,
-            'analytic_tag_ids': [Command.set(self.analytic_tag_ids.ids)],
+            'analytic_tag_ids': [Command.set(self.order_id.order_line.mapped('analytic_tag_ids').ids)],
```

Comment:
@ppr-odoo there's an issue here when multiple project are created from a single SO
all the project will get all the tags